.github/workflows/ci-test.yml

```diff
@@ -56,6 +56,22 @@ jobs:
         run: |
           ./contribution/k3s/install_k3s.sh
 
+      - name: Setup local registry
```

Comment:
let's directly use YAML files in [annotations](https://github.com/kubearmor/KubeArmor/tree/main/deployments/annotations).